modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -933,14 +907,9 @@ public String toDefaultValue(Schema schema) {
                 return null;
             }
 
-            String typeDeclaration = String.format(Locale.ROOT, "String, %s", getTypeDeclaration(getAdditionalProperties(schema)));
-            Object java8obj = additionalProperties.get("java8");
-            if (java8obj != null) {
-                Boolean java8 = Boolean.valueOf(java8obj.toString());
-                if (java8 != null && java8) {
-                    typeDeclaration = "";
-                }
-            }
+            // comment out below for JDK7
```

Comment:
Same - can be removed?

Reply:
sure, will remove those in one of the upcoming PRs

Reply:
Done via 1892f2f9931